THCalendarDatePickerExample/THCalendarDatePickerExample/TestViewController.m

```diff
@@ -46,6 +45,7 @@ - (IBAction)touchedButton:(id)sender {
     [self.datePicker setSelectedBackgroundColor:[UIColor colorWithRed:125/255.0 green:208/255.0 blue:0/255.0 alpha:1.0]];
     [self.datePicker setCurrentDateColor:[UIColor colorWithRed:242/255.0 green:121/255.0 blue:53/255.0 alpha:1.0]];
     [self.datePicker setCurrentDateColorSelected:[UIColor yellowColor]];
+    self.datePicker.date = self.curDate;
```

Comment:
What setting? I've moved setting of the date property to the end to make it work properly.